src/docs/Area.component.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const Area = styled.div`
+	display: flex;
+	flex-basis: 100%;
+	align-items: center;
+	justify-content: center;
+	margin: 1rem;
+	padding: 0.5rem;
+	min-height: 0;
+	font-weight: bold;
+	font-size: 2rem;
+	color: coral;
+	background: cornsilk;
+	border: 1px dashed coral;
+	border-radius: 1rem;
```

Comment:
These could also be tokens, right? 
They could also benefit from being pixels rather than rems: they are rarely meant to change and stretch with the initial font size.